README.md

```diff
@@ -1,36 +1,121 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Cadence Web UI
+
+[![Build Status](https://github.com/uber/cadence-web/actions/workflows/build.yml/badge.svg)](https://github.com/uber/cadence-web/actions/workflows/build.yml) [![Docker Status](https://github.com/uber/cadence-web/actions/workflows/docker_publish.yml/badge.svg)](https://hub.docker.com/r/ubercadence/web/tags)
+
+Cadence is a distributed, scalable, durable, and highly available orchestration engine we developed at Uber Engineering to execute asynchronous long-running business logic in a scalable and resilient way.
+
+This web UI is used to view workflows from [Cadence][cadence], see what's running, and explore and debug workflow executions.
+
 
 ## Getting Started
 
-First, run the development server:
+### Configuration
+
+Set these environment variables if you need to change their defaults
+
+| Variable                     | Description                                  | Default          |
+| ---------------------------- | -------------------------------------------- | ---------------- |
+| CADENCE_GRPC_PEERS           | Comma-delimited list of gRPC peers            | 127.0.0.1:7833   |
+| CADENCE_GRPC_SERVICES_NAMES  | Comma-delimited list of gRPC services to call | cadence-frontend |
+| CADENCE_CLUSTERS_NAMES       | Comma-delimited list of cluster names         | cluster0         |
+| CADENCE_WEB_PORT             | HTTP port to serve on                        | 8088             |
+| ENABLE_AUTH                  | Enable auth feature                          | false            |
+| CADENCE_ADMIN_SECURITY_TOKEN | Admin token for accessing admin methods      | ''               |
+
+Note: To connect `cadence-web` to multiple clusters, you will need to add comma-delimted entries for `CADENCE_GRPC_PEERS`, `CADENCE_GRPC_SERVICES_NAMES` & `CADENCE_CLUSTERS_NAMES` for each cluster (each cluster values are grouped by their index within the Comma-delimited lists).
+
+Example:
+```
+CADENCE_GRPC_PEERS=127.0.0.1:3000,127.0.0.1:5000 
+CADENCE_GRPC_SERVICES_NAMES=cadence-frontend-cluster0,cadence-frontend-cluster1
+CADENCE_CLUSTERS_NAMES=cluster0,cluster1
+```
+
+
+### Using cadence-web
+
+Latest version of `cadence-web` is included in `cadence` composed docker containers that can be found [here][cadence]. To start using it clone `cadence` repo and run the following command
```

Comment:
```suggestion
The latest version of `cadence-web` is included in the `cadence` composed docker containers in the [main Cadence repository][cadence]. Follow the instructions there to get started.
```
You can remove the docker-compose command here as well, users can follow the server repo readme for this.